<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryForm</class>
 <widget class="QWidget" name="HistoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="device_combobox_">
       <item>
        <property name="text">
         <string>设备1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>设备2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_data_button_">
       <property name="text">
        <string>加载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wipe_data_button_">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="data_table_view_"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
